{
  "command": "7",
  "error_type": "AttributeError",
  "error_message": "'ConnectionConfig' object has no attribute 'connection_mode'",
  "traceback": "Traceback (most recent call last):\n  File \"/Users/liuweigang/sgzAss/src/cli/game_cli.py\", line 109, in start_interactive_mode\n    self._handle_config()\n  File \"/Users/liuweigang/sgzAss/src/cli/game_cli.py\", line 468, in _handle_config\n    print(f\"  连接模式: {self.config.connection.connection_mode}\")\nAttributeError: 'ConnectionConfig' object has no attribute 'connection_mode'\n",
  "timestamp": "2025-07-29 17:52:36",
  "component": "GameCLI",
  "version": "3.0"
}